app/src/main/java/com/kanshu/kanshu/LoginActivity.java

```diff
@@ -35,4 +56,35 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
         return super.onOptionsItemSelected(item);
     }
+
+    public void Login(View v){
+
+        HttpGet request = new HttpGet("https://kanshu-ds.herokuapp.com/login");
```

Comment:
I would think there would be a more Retrofit way of doing this:

e.g. 
http://futurestud.io/blog/android-basic-authentication-with-retrofit/
http://stackoverflow.com/questions/21370620/retrofit-post-request-w-basic-http-authentication-cannot-retry-streamed-http